### PR TITLE
Add suffixes to typos from "pa" to "pl"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -30103,30 +30103,43 @@ pacakging->packaging
...
 pacht->patch
 pachtches->patches
 pachtes->patches
 pacjage->package
 pacjages->packages
 packacge->package
+packacged->packaged
+packacges->packages
+packacging->packaging
+packae->package
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:30122]: Depending on the context of course, I would interpreted `packae` as `pancake`.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -30954,7 +31066,11 @@ peroidicity->periodicity
 peroids->periods
 peronal->personal
 peroperly->properly
+perosn->person
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31069]: Perhaps plural `perosns->persons`. Typo found on internet pages.

**Review by DimitriPapadopoulos — COMMENTED** [review]
I am wondering whether you have scripts to help you. Once we finish this run of fixes, you might consider contributing them.

**Comment by int-y1** [comment]
> I am wondering whether you have scripts to help you. Once we finish this run of fixes, you might consider contributing them.

Yes, I have a script that created this list of typos. I'm not sure if my script should be added to the repo. Once this round is done, I might start another round, or create a new issue with a typo list.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**